ds-streamlit/files/requirements.txt

```diff
@@ -0,0 +1,5 @@
+streamlit==1.17.*
```

Comment:
I suggest splitting requirements into app requirements and test requirements, see for example how it is done in https://github.com/opendevstack/ods-quickstarters/tree/master/be-python-flask/files